website/docs/language/style.mdx

```diff
@@ -678,4 +678,4 @@ For more information, refer to the [policy enforcement documentation](/terraform
 
 This article introduces some considerations to keep in mind as you standardize your organization's Terraform style guidelines. Enforcing a standard way of writing and organizing your Terraform code across your organization ensures that it is readable, maintainable, and shareable.
 
-The [HashiCorp Well-Architected Framework](/well-architected-framework) provides more guidance on [adapting your Terraform workflows for scale](/well-architected-framework/operational-excellence/operational-excellence-terraform-maturity).
+The [HashiCorp Well-Architected Framework](/well-architected-framework) provides more guidance on [adapting your Terraform workflows for scale](/well-architected-framework/operational-excellence/operational-excellence-terraform-maturity). To learn more Terraform adoption best practices, refer to the [Phases of Terraform adoption](/terraform/intro/phases) documentation.
```

Comment:
```suggestion
The [HashiCorp Well-Architected Framework](/well-architected-framework) provides more guidance on [adapting your Terraform workflows for scale](/well-architected-framework/operational-excellence/operational-excellence-terraform-maturity). To learn more Terraform adoption best practices, refer to [Phases of Terraform adoption](/terraform/intro/phases).
```

Reply:
should the operational excellence link be removed? i thought that was the article we're relocating

Reply:
Good catch, I'll get that removed!